mods/ts/rules/civilian-infantry.yaml

```diff
@@ -270,13 +272,16 @@ VISC_LRG:
 		HP: 500
 	Armament:
 		Weapon: SlimeAttack
+		FireDelay: 10
```

Comment:
Can you link to a replay that shows off these critters attacking on the latest version of this branch?  If it desyncs then we know its a bug.